bootstrap.inc.sh

```diff
@@ -101,6 +101,7 @@ function _bootstrap_configure_common() {
     KeymanHosts.php
     KeymanSentry.php
     MarkdownHost.php
+    ImageRabdomizer.php
```

Comment:
typo?

Reply:
Haha, it is. Thank you! 
Also @mcdurdin, how do I get this to build (./build.sh configure) on my keyman.localhost.com? 

I tried changing this: `readonly BOOTSTRAP_VERSION=chore/v1.0` but error: `./build.sh: line 14: _common/keyman-local-ports.inc.sh: No such file or directory`. I'm confused and stuck.

Reply:
Try:
 
```
readonly BOOTSTRAP_VERSION=chore/v0.10
```